### PR TITLE
Feature/Add Sharepoint graphSearchQuery function

### PR DESCRIPTION
**Pull request by msalvatti**
Add Sharepoint graphSearchQuery function

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-frontify@1.0.2-canary.217.dc55463.0
  npm install @friggframework/api-module-microsoft-sharepoint@0.0.8-canary.217.dc55463.0
  # or 
  yarn add @friggframework/api-module-frontify@1.0.2-canary.217.dc55463.0
  yarn add @friggframework/api-module-microsoft-sharepoint@0.0.8-canary.217.dc55463.0
  ```
</details>

### CODE REVIEW

File: api-module-library/frontify/api.test.js [hunk 1]
```diff
@@ -1493,6 +1493,8 @@ describe(`${Config.label} API Tests`, () => {
                 let scope;
 
                 beforeEach(() => {
+
+                    console.log(buildQl())
```
Comment (leofmds) [inline api-module-library/frontify/api.test.js:1497]: Remove log

File: api-module-library/sharepoint/api.js [hunk 2]
```diff
@@ -127,6 +127,36 @@ class Api extends OAuth2Requester {
         return response;
     }
 
+    async graphSearchQuery(query) {
```
Comment (leofmds) [inline api-module-library/sharepoint/api.js:130]: Can you please add a test for this function?

**Review by leofmds — CHANGES_REQUESTED** [review]
Looks good, just add a missing test to the function and remove unnecessary log

**Review by leofmds — APPROVED** [review]
👍🏻

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-frontify@1.0.2`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-frontify%401.0.2), [`@friggframework/api-module-microsoft-sharepoint@0.0.8`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-microsoft-sharepoint%400.0.8) :rocket:

**Pull request merged**